### ISSUE TITLE
go to definition fail: Source.actorRef

### ISSUE THREAD

**Issue by fommil** [ensime/ensime-server#1041]
from Akka Stream.

Need to investigate why.

**Comment by fommil** [comment]
in this line, we get epic fails:

```scala
        val source = Source.actorRef[RpcResponseEnvelope](0, OverflowStrategy.fail)
```

we can't jump to source for `Source`, `actorRef`, `RpcResponseEnvelope`, `OverflowStrategy` or `fail`. Wow.

**Comment by fommil** [comment]
I think the presentation compiler is just struggling with all the spray pimping going on. I see this a lot in the logs

```
symbolAt for class scala.reflect.internal.Trees$EmptyTree$: <empty>
```

**Comment by edani** [comment]
It's working for me. It may have been fixed by #1058
